gwpy/table/io/gravityspy.py

```diff
@@ -92,10 +92,14 @@ def get_gravityspy_triggers(tablename, engine=None, **kwargs):
 
 def get_connection_str(db='gravityspy',
                        host='gravityspy.ciera.northwestern.edu',
-                       user=os.getenv('GRAVITYSPY_DATABASE_USER', None),
-                       passwd=os.getenv('GRAVITYSPY_DATABASE_PASSWD', None)):
+                       user=None,
+                       passwd=None):
```

Comment:
@scottcoughlin2014, can you add a docstring for this method that notes how variables are resolved from `os.environ` if not given?

Reply:
Ya I can do this

Reply:
@duncanmmacleod I have attempted to add more helpful doc strings let me know what you think